release.config.js

```diff
@@ -0,0 +1,62 @@
+module.exports = {
+    branches: ['master'],
+    tagFormat: 'v${version}',
+    repositoryUrl:
+        'https://github.com/mparticle-integrations/mparticle-javascript-integration-leanplum',
+    plugins: [
+        [
+            '@semantic-release/commit-analyzer',
+            {
+                preset: 'angular',
+                releaseRules: [
+                    { type: 'feat', release: 'minor' },
+                    { type: 'ci', release: 'patch' },
+                    { type: 'fix', release: 'patch' },
+                    { type: 'docs', release: 'patch' },
+                    { type: 'test', release: 'patch' },
+                    { type: 'refactor', release: 'patch' },
+                    { type: 'style', release: 'patch' },
+                    { type: 'build', release: 'patch' },
+                    { type: 'chore', release: 'patch' },
+                    { type: 'revert', release: 'patch' },
+                ],
+            },
+        ],
+        [
+            '@semantic-release/release-notes-generator',
+            {
+                preset: 'angular',
+            },
+        ],
+        [
+            '@semantic-release/changelog',
+            {
+                changelogFile: 'CHANGELOG.md',
+            },
+        ],
+        ['@semantic-release/npm'],
+        [
+            '@semantic-release/exec',
+            {
+                prepareCmd: 'sh ./scripts/release.sh',
```

Comment:
This file does not exist.  We should also pull this from here:
https://github.com/mparticle-integrations/mparticle-javascript-integration-google-analytics-4/blob/master/packages/GA4Client/scripts/release.sh